lib/plugins/package/lib/packageService.js

```diff
@@ -23,6 +23,17 @@ module.exports = {
     return _.union(packageIncludes, include);
   },
 
+  getRuntime(runtime) {
+    const defaultRuntime = 'nodejs12.x';
+    return (
+      runtime ||
+      this.options.runtime ||
+      this.serverless.config.runtime ||
```

Comment:
Same concerns apply here.

(Otherwise it's a good call, to unify runtime resolution)